hierr_test.go

```diff
@@ -5,33 +5,23 @@ import "fmt"
 
 func ExampleError() {
 	testcases := []error{
-		Errorf(nil, ""),
 		Errorf(nil, "simple error"),
-		Errorf(nil, "integer: %d", 1),
 		Errorf(errors.New("nested"), "top level"),
 		Errorf(errors.New("nested"), "top level: %s", "formatting"),
-		Errorf(Errorf(errors.New("low level"), "nested"), "top level"),
+		Errorf(Errorf(errors.New("low level"), "nested"), "top level integer %d", 1),
 	}
 
 	for _, test := range testcases {
 		fmt.Println()
 		fmt.Println("{{{")
-		fmt.Println(test.Error())
+		fmt.Println(test)
 		fmt.Println("}}}")
 	}
 
 	// Output:
 	//
 	// {{{
-	//
```

Comment:
@kovetskiy: I guess it's better to separate examples (one per expected output), but otherwise it's quite readable (more readable, than comparison to ugly multiline strings).